scripts/qa_crowbarsetup.sh

```diff
@@ -4439,6 +4439,12 @@ function oncontroller_testpostupgrade
 {
     # retrieve the ping results
     local fips=$(openstack floating ip list -f value -c "Floating IP Address")
+
+    # remove manila-service fip from list
+    manila_vm=$(openstack server list --all-projects -f value | grep manila-service | awk '{ print $1 }' )
```

Comment:
I tried to do a `openstack server show` passing the name, but only worked for me passing the ID, that's why I wrote it this way.